README.md

```diff
@@ -82,17 +79,29 @@ parsing of the URL.
 
 # Examples
 
-Download a single file:  
-**wcurl example.com/filename.txt**
+* Download a single file:
+
+  `wcurl example.com/filename.txt`
+
+* Download two files in parallel:
+
+  `wcurl example.com/filename1.txt example.com/filename2.txt`
+
+* Download a file passing the _--progress-bar_ and _--http2_ flags to curl:
+
+  `wcurl --curl-options="--progress-bar --http2" example.com/filename.txt`
+
+* Resume from an interrupted download (if more options are used, this needs to be the last one in the list):
+
+  `wcurl --curl-options="--continue-at -" example.com/filename.txt`
 
-Download two files in parallel:  
-**wcurl example.com/filename1.txt example.com/filename2.txt**
+<a name="testsuite"></a>
 
-Download a file passing the _--progress-bar_ and _--http2_ flags to curl:  
-**wcurl --curl-options="--progress-bar --http2" example.com/filename.txt**
+# Running the testsuite
 
-Resume from an interrupted download (if more options are used, this needs to be the last one in the list):  
-**wcurl --curl-options="--continue-at -" example.com/filename.txt**
+If you would like to run the tests, you will first need to install the
+`shunit2` package.  On Debian-like and Fedora-like systems, the
+package is called `shunit2`.
```

Comment:
Maybe add another line for how you then actually run those said tests?

Reply:
Good point.  Done.